lint381/c.py

```diff
@@ -11,9 +11,17 @@
 
 
 @linter.register
+@with_matched_tokens(start=match_regex("^sizeof$"), end=match_regex("^char$"))
+def prohibited_unsigned_char(source, *, match):
+    """sizeof char should be avoided as the C standard defines it to be a 1."""
+    yield Error("sizeof(char) should not be used as it is defined by the C "
+                "standard to be equal to 1", tokens=[match[0]])
+
+
+@linter.register
 @with_matched_tokens(start=match_regex("^(unsigned|float)$"))
 def prohibited_types(source, *, match):
-    """Flag prohibited numeric types."""
+    """flag prohibited numeric types."""
```

Comment:
The convention in the codebase is to document with real sentences, including capitalization. So revert this bit.